doc/src/concepts/components.md

```diff
@@ -76,6 +76,7 @@ details.
 [build-std]: https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#build-std
 [miri]: https://github.com/rust-lang/miri/
 [RLS]: https://github.com/rust-lang/rls
+[rustdoc]: https://github.com/rust-lang/rust/tree/master/src/tools/rustdoc
```

Comment:
I think this should link to the rustdoc documentation at https://doc.rust-lang.org/rustdoc/

Reply:
I'm not sure exactly what links need to be used here, if the emphasis is on internal tools then the current links might be better? If the emphasis is on rusctdoc itself maybe a link to the documentation would be better?

Reply:
The intent here is to provide information for a user of rustup to learn what each component is.  Some of these (like miri or rustfmt) link to github because they don't have any other documentation.

Reply:
Changed. Thanks for your review!